public/docs/PrivacyPolicyEn.html

```diff
@@ -11,7 +11,7 @@
         <h1 class="font-sans flex-1 text-left md:text-xl font-extrabold uppercase">
             Privacy policy
         </h1>
-        <a class="items-center button flex-4 w-24 mr-1 border text-white dark:border-dark-5" onclick="window.history.back()">
+        <a class="items-center button flex-4 w-24 mr-1 border text-white dark:border-dark-5" href="https://dataocean.us/" target="_blank">
```

Comment:
what if you open this page from platform? (dp.dataocean.us)
![image](https://user-images.githubusercontent.com/38906004/106571293-a3c6ec80-653f-11eb-9955-d774e183fa4b.png)

Reply:
You ask about "terms an conditions", but task about "privacy policy".
Should I make the same with "terms"?

Reply:
Is there a difference?

I think to remove the back button

Reply:
This question or command to execute?
Actually, I totally agree, that this button is useless, because in first sentence available links to both pages (platform and landing), and new tab is make impossible to return to previous.